File: D1-Morning/C#/D1-S2/W13-OneToOne/W13-OneToOne/Address.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace W13_OneToOne
{
    public class Address
    {
        Address()
        {
        }

        public Address(string aptNo, string house, string road, string area, string thana, string district)
        : this()
        {
            AptNo = aptNo;
            House = house;
            Road = road;
            Area = area;
            Thana = thana;
            District = district;
        }
        public string AptNo { set; get; }
        public string House { set; get; }
        public string Road { set; get; }
        public string Area { set; get; }
        public string Thana { set; get; }
        public string District { set; get; }
    }
}

File: D1-Morning/PHP/D1-S2-PHP/W13-OneToOne/Address.php
<?php


class Address
{
    public $aptNo;
    public $house;
    public $road;
    public $area;
    public $thana;
    public $district;

    public function __construct($aptNo, $house, $road, $area, $thana, $district)
    {
        $this->aptNo = $aptNo;
        $this->house = $house;
        $this->road = $road;
        $this->area = $area;
        $this->thana = $thana;
        $this->district = $district;
    }


}